client/devdocs/index.js

```diff
@@ -82,7 +82,7 @@ export default function() {
 
 		if ( config.isEnabled( 'devdocs/gutenberg-blocks' ) ) {
 			page(
-				'/devdocs/gutenberg-blocks',
+				'/devdocs/gutenberg-blocks/:block*',
```

Comment:
Hmm, what does wildcard here mean for the `:block*`? The pagejs documentation is a little lacking. I know that `:block?` would make this match optionally with or without the block name here.

Reply:
`*` is used as a wildcard because `?` doesn't match params with slashes (even if they are encoded): https://github.com/visionmedia/page.js/issues/187